package.json

```diff
@@ -2,7 +2,11 @@
 	"name": "plop",
 	"version": "2.6.0",
 	"description": "Micro-generator framework that makes it easy for an entire team to create files with a level of uniformity",
-	"main": "./src/plop",
+	"main": "./dist/plop",
+	"files": [
+		"bin/",
+		"dist/"
+	],
```

Comment:
Shrink that package!

```
npm notice package size:  427.8 kB
npm notice unpacked size: 1.2 MB
```

Reply:
I'd love to leave `README.md` and other docs included in this, personally. I don't know off the top of my head how "files" works, would this need to be modified to allow for that?

Reply:
Files automatically includes the LICENSE, README, and package.json. Here's a preview of the publish result


```bash
npm notice === Tarball Contents ===
npm notice 1.1kB  LICENSE
npm notice 171B   dist/index.js
npm notice 293B   bin/plop.js
npm notice 1.1MB  dist/plop.js
npm notice 1.1kB  package.json
npm notice 26.4kB README.md
npm notice 10.3kB dist/plop.js.LICENSE.txt
npm notice === Tarball Details ===
npm notice name:          plop
npm notice package size:  428.0 kB
npm notice unpacked size: 1.2 MB
npm notice total files:   7
```